CORDOVA_UPGRADE_GUIDE.md

```diff
@@ -0,0 +1,235 @@
+# Upgrade Guide To The New Branch PhoneGap/Cordova SDK
+
+In the effort of making the quickly growing functionality of our Web SDK, available to your PhoneGap or Cordova apps, we've merged the two together. This will require some very simple changes to the implementation, but you'll gain a number of very useful functions and the Branch promise of continuous updates and improvements!
```

Comment:
i guess that's not that different from what you have now, but it just makes it clearer... if you think it'd be good let's do it; if not, let's not waste time